### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dkorts1**
minor changes, grammar, and typos

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ If you run Mailtrain on a VPS, you will have to find the PTR configuration somew
 
 ## Upgrade
 
-  * Replace old files with new ones by running in the Mailtrain folder `git pull origin master` if you used Git to set Mailtrain up or just download [new files](https://github.com/Mailtrain-org/mailtrain/archive/master.zip) and replace old ones with these
+  * Replace old files with new ones by running  in the Mailtrain folder ` if you used Git to set Mailtrain up or just download [new files](https://github.com/Mailtrain-org/mailtrain/archive/master.zip) and replace old ones with these
```
Comment (bures) [inline docs/README.md]: It seems that the  `git pull origin master` disappeared from here. Was it intentional?

File: docs/README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Check out [ZoneMTA](https://github.com/zone-eu/zone-mta) as an alternative self
 
 ## Cons
 
-  * Beta-grade software. Might or might not work as expected. There are several users with list sizes between 100k and 1M and Mailtrain seems to work for them but YMMV
```
Comment (bures) [inline docs/README.md]: Why has the sentence about list sizes taken out?

**Comment by dkorts1** [comment]
Hello
1. I am not sure what disappeared, it seems it was not intentional
    I did delete my own previous request, as it was the same as the latest
one before  I added several more edits.
2. The sentence seemed not clear to me, perhaps I should have just changed
it to: *" **Beta-grade software. Several users reported success with lists
of various sizes (from 100k to 1M) however there is no absolute guarantee
it will always work as expected*."
I had to google "YMMV", which is a US (slang) term, seems to me these
abbreviations are not really for a document.
thanks

**Comment by dkorts1** [comment]
ps- I see now what you asked regarding the part that disappeared, that was
not intentional

**Comment by bures** [comment]
Perfect. I included the suggestion about beta-grade software, added the code snippet that disappeared, and merged it. Thanks.

**Pull request merged**